app/src/main/java/de/xikolo/models/migrate/RealmSchemaMigration.java

```diff
@@ -121,7 +121,7 @@ public void migrate(DynamicRealm realm, long oldVersion, long newVersion) {
 
             oldVersion++;
         }
-
+        
```

Comment:
We don't need these whitespaces in the diff 😅

Reply:
I have no idea why they're in here. In an ancient commit, I accidentally deleted the line, then I fixed it in another commit. As I understand it Github shows that a line has been taken away, but another has been added. Is this my fault or Githubs?

Reply:
Probably the IDE added the whitespaces, pls remove them and keep the empty line.

Reply:
Solved